### ISSUE TITLE
CockroachDB index schema changed in v2+

### ISSUE THREAD

**Issue by stanislas-m** [gobuffalo/fizz#84]
Running the tests on pop with the latest fizz version shows an hidden issue in fizz:

```
Error: error processing testdata/migrations/20181104140855_books_description.up.fizz: could not fizz the migration testdata/migrations/20181104140855_books_description.up.fizz: line 0: could not call change_column function: sql: Scan error on column index 1, name "non_unique": sql/driver: couldn't convert "NO" into type bool
```

It seems CockroachDB index schema changed in v2+:

v1.1 docs:
> NON_UNIQUE | false if the index was created by a Unique constraint; true if the index was not created by a Unique constraint.

v2.0 docs:
> non_unique | NO if the index was created with the UNIQUE constraint; YES if the index was not created with UNIQUE.

We need to either drop support for v1, or add a check for the version.

See https://github.com/gobuffalo/pop/pull/519/checks?check_run_id=457147696

**Comment by mclark4386** [comment]
Would probably simplify things if we dropped v1, but am happy to add conditionals in there... v2 isn't even supported by cockroach labs anymore, so letting go of v1 is probably safe.

**Comment by stanislas-m** [comment]
Fixed with #85.

**Comment by stanislas-m** [comment]
Awesome, thanks!
